src/main/java/frc/robot/Constants.java

```diff
@@ -3,10 +3,13 @@
 public final class Constants {
 
     public static final int DRIVER_CONTROLLER = 0;
-
-    public final static int LEFT_FRONT_ID = 1;
-    public final static int LEFT_BACK_ID = 2;
-    public final static int RIGHT_FRONT_ID = 3;
-    public final static int RIGHT_BACK_ID = 4;
+    public static final int OPERATOR_CONTROLLER = 1;
+
+    public final static int LEFT_FRONT_ID = 2;
```

Comment:
Why are we starting the first ID on 2? It should probably start on 1 or 0. 1 might be better but we should not start on 2